src/BioFSharp/Digestion.fs

```diff
@@ -21,15 +22,11 @@ module Digestion =
          {Name = name; Expression = f}
 
     /// Digested peptide
-    // TODO: type of ProteinID to 'a; rename "missCleavageStart" to "cleavageStart"; Same for ..End..
     type DigestedPeptide<'a> = {
-        ///Identifier of protein
         ProteinID: 'a
-        ///
         MissCleavages: int
         CleavageStart:int
         CleavageEnd: int
-        ///Sequence of peptide
         PepSequence: AminoAcid list
```

Comment:
Genuine question: is AminoAcidList an artifact before BioList existed? I guess this can be changed to BioList<AminoAcid> without breaking previous versions. What do you think?

Reply:
Moved to #95